tactics/declare.mli

```diff
@@ -59,9 +59,12 @@ val definition_entry
   : ?fix_exn:Future.fix_exn
   -> ?opaque:bool
   -> ?inline:bool
+  -> ?feedback_id:Stateid.t
+  -> ?section_vars:Id.Set.t
   -> ?types:types
   -> ?univs:Entries.universes_entry
   -> ?eff:Evd.side_effects
+  -> ?univc:Univ.ContextSet.t
```

Comment:
There should be a comment explaining this argument

Reply:
Done, I'm not sure I fully grasp how body-only universe constraints are supposed to work yet so please correct me.

By the way, about the naming of arguments I don't have a strong opinion; all that I'm doing is making the _uniform_, but feel free to suggest a different naming scheme.

Reply:
Maybe `univsbody`

Reply:
Done.